x-pack/plugins/license_management/__jest__/__snapshots__/upload_license.test.js.snap

```diff
@@ -313,7 +313,7 @@ exports[`UploadLicense should display a modal when license requires acknowledgem
                         maxWidth={true}
                         onClose={[Function]}
                       >
-                        <Component
+                        <_default
```

Comment:
We use https://github.com/davidtheclark/focus-trap-react, which doesn't declare `displayName`. Seems it's a result of babel transpilation referring to export type.